### PR TITLE
Add documentation: creating a document via a command

### PR DESCRIPTION
**Pull request by szz**
history: https://sulu-io.slack.com/archives/C0430GGCV/p1634221207268000

| Q | A
| --- | ---
| Fixed tickets | 
| Related PRs | 
| License | MIT

#### What's in this PR?

Add documentation: creating a document via a command

### CODE REVIEW

File: reference/components/document-manager/using_the_document_manager.rst [hunk 1]
```diff
@@ -62,6 +62,14 @@ on which subscribers you have registered.
 
 See the :doc:`subscribers` chapter for more information.
 
+.. note::
+
+    To create a document via a Sulu CLI command (e.g., import from an external source), run the command within the admin context.
+
+    .. code-block:: bash
+
+        bin/adminconsole your:command
```
Comment (alexander-schranz) [inline reference/components/document-manager/using_the_document_manager.rst]: This text should be more general as it need to be the `admin` context. So you can not create a document inside a website controller you will need.

So something like "Documents can only be created in the admin context. For cli this means using the the `bin/adminconsole` and for controller it is required to have run them in the `/admin` context or using a symfony messenger to create them asynchron via admin CLI."

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

**Review by alexander-schranz — CHANGES_REQUESTED** [review]
@szz Thank youf or the pull request! As this is not only for `2.x` we should target here atleast the `2.3` branch which is the current lowest supported version of sulu 2.

File: reference/components/document-manager/using_the_document_manager.rst (hunk 1 above)
Comment (Prokyonn) [reply reference/components/document-manager/using_the_document_manager.rst]:
```suggestion
    Be aware that documents can only be created in the admin context, not in the website context. For CLI commands, this means you must use ﻿``bin/adminconsole`` to execute the command. Alternatively, you can execute a command that dispatches messages via the Symfony Messenger. In this case, the consumer that consumes the dispatched message must also run in the admin context.

    .. code-block:: bash

        bin/adminconsole your:command
```

**Review by Prokyonn — APPROVED** [review]
@alexander-schranz Please verify the adapted text :)

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
@szz thank you!

**Pull request merged**